apps/examples/wc-next13/styles/globals.css

```diff
@@ -0,0 +1,107 @@
+:root {
```

Comment:
For demo/testing webcomponents context I believe we don't get any benefit by commiting this stylesheet.